releases.json

```diff
@@ -108,6 +108,15 @@
       "6.2.2-1"
     ]
   },
+  "cli11": {
+    "dependency_names": [
+      "cli11"
+    ],
+    "program_names": [],
```

Comment:
You can omit empty program_names list now, I removed them everywhere else recently.

Reply:
If you can just fix this small nitpicking, I think it's ready to be merged now. Thanks!